tasks/destroy.py

```diff
@@ -17,7 +17,7 @@ def destroy(
     scenario_name: str,
     config_path: Optional[str] = None,
     stack: Optional[str] = None,
-    force_yes: Optional[bool] = False,
```

Comment:
If we set the default to `True` I think we should rather remove this option. I do not think it makes sense to have a `--no-force-yes` option